diff -urpN klippy-stock/extras/shaper_calibrate.py klippy/extras/shaper_calibrate.py
--- klippy-stock/extras/shaper_calibrate.py	2025-01-16 00:15:20.662822971 -0500
+++ klippy/extras/shaper_calibrate.py	2025-01-16 00:07:08.182912776 -0500
@@ -9,7 +9,7 @@ import math
 import multiprocessing
 import traceback
 
-shaper_defs = importlib.import_module(".shaper_defs", "extras")
+shaper_defs = importlib.import_module(".shaper_defs", "klippy.extras")
 
 MIN_FREQ = 5.0
 MAX_FREQ = 200.0
diff -urpN klippy-stock/klippy.py klippy/klippy.py
--- klippy-stock/klippy.py	2025-01-16 00:15:20.392823026 -0500
+++ klippy/klippy.py	2025-01-16 00:13:29.232843563 -0500
@@ -138,6 +138,7 @@ class Printer:
             return self.objects[section]
         module_parts = section.split()
         module_name = module_parts[0]
+
         extras_py_name = os.path.join(
             os.path.dirname(__file__), "extras", module_name + ".py"
         )
@@ -174,18 +175,18 @@ class Printer:
 
         if found_in_plugins:
             mod_spec = importlib.util.spec_from_file_location(
-                "extras." + module_name, plugins_py_name
+                "klippy.extras." + module_name, plugins_py_name
             )
             mod = importlib.util.module_from_spec(mod_spec)
             mod_spec.loader.exec_module(mod)
         elif found_in_plugins_dir:
             mod_spec = importlib.util.spec_from_file_location(
-                "plugins." + module_name, plugins_py_dirname
+                "klippy.plugins." + module_name, plugins_py_dirname
             )
             mod = importlib.util.module_from_spec(mod_spec)
             mod_spec.loader.exec_module(mod)
         else:
-            mod = importlib.import_module("extras." + module_name)
+            mod = importlib.import_module("klippy.extras." + module_name)
 
         init_func = "load_config"
         if len(module_parts) > 1:
@@ -431,7 +432,7 @@ def import_test():
                 if not os.path.exists(iname):
                     continue
                 module_name = fname
-            importlib.import_module(mname + "." + module_name)
+            importlib.import_module("." + mname + "." + module_name, package="klippy")
     sys.exit(0)
 
 
diff -urpN klippy-stock/toolhead.py klippy/toolhead.py
--- klippy-stock/toolhead.py	2025-01-16 00:15:20.372823022 -0500
+++ klippy/toolhead.py	2025-01-16 00:14:53.672827769 -0500
@@ -314,7 +314,7 @@ class ToolHead:
         self.extruder = extruder.DummyExtruder(self.printer)
         kin_name = config.get("kinematics")
         try:
-            mod = importlib.import_module("kinematics." + kin_name)
+            mod = importlib.import_module(".kinematics." + kin_name, package='klippy')
             self.kin = mod.load_kinematics(self, config)
         except config.error as e:
             raise

